internal/entity/common.go

```diff
@@ -178,8 +194,17 @@ func NewSeverity(url string) Severity {
 }
 
 type Cvss struct {
+	Info
 	Vector        string
 	Base          *metric.Base
 	Temporal      *metric.Temporal
 	Environmental *metric.Environmental
 }
+
+type Info struct {
+	CreatedAt time.Time `json:"created_at"`
+	CreatedBy string    `json:"created_by"`
+	UpdatedAt time.Time `json:"updated_at"`
+	UpdatedBy string    `json:"updated_by"`
+	DeletedAt time.Time `json:"deleted_at,omitempty"`
+}
```

Comment:
I like the way this is solved, though I found "Info" naming to vague.

I'd prefer something more concrete that describes the type of information stored, something like:

- LifecycleMetadata
- LifecycleDetails
- AuditDetails
- AuditRecord
- AuditMetadata

@dorneanu @michalkrzyz @MR2011  What do you folks think?

Reply:
This will be integrated into every model/entity so we could use Model or Entity as name. [Gorm ](https://gorm.io/docs/models.html#gorm-Model) uses the same approach.

Reply:
We agreed on Metadata